src/schema/entity.rs

```diff
@@ -211,6 +211,9 @@ where
     if let Some(value) = orm_column_def.default_value {
         column_def.default(value);
     }
+    if let Some(value) = orm_column_def.extra {
+        column_def.default(value);
+    }
```

Comment:
Found a typo

```suggestion
    if let Some(value) = orm_column_def.extra {
        column_def.extra(value);
    }
```

Reply:
Hi @billy1624 thank you for your comment,
what about replace with `column_def.default(SimpleExpr::CustomWithExpr(value, vec![]));`, I found the generated default expr in schema is going to quoted with `'` in this(`column_def.default(value);`) way.